### PR TITLE
Add label for enter room field

### LINKED ISSUE #5739: Add label to room name
**Is your feature request related to a problem you are facing?**
Based on the discussion at https://community.jitsi.org/t/accessibility-ranking-of-jitsi/27796, I would like to see a label for the room name, in order to help users to identify what the input field is

**Describe the solution you'd like**
Add a <label> before the input field.
https://github.com/jitsi/jitsi-meet/blob/9fdc18d1ec41707be3fc1f1de06b81c7f1394fb9/react/features/welcome/components/WelcomePage.web.js#L195

I will add a pull request for this.

### PR DESCRIPTION
**Pull request by guerda**
Improve accessibility, see #5739

Close #5739

### CODE REVIEW

File: react/features/welcome/components/WelcomePage.web.js [hunk 1]
```diff
@@ -198,6 +198,7 @@ class WelcomePage extends AbstractWelcomePage {
                                 { t('welcomepage.enterRoomTitle') }
                             </div>
                             <form onSubmit = { this._onFormSubmit }>
+                                <label htmlFor = 'enter_room_field'>Room name:</label>
```
Comment (luixxiul) [inline react/features/welcome/components/WelcomePage.web.js:201]: This should be hopefully localizable.

See: https://github.com/jitsi/jitsi-meet/blob/82b9cb77447a74ef6812b35b9e46ec1971b0e5ce/react/features/welcome/components/WelcomePage.web.js#L188

Also: https://github.com/jitsi/jitsi-meet/tree/3f679407a93d033a2a99824d06bfe806eae8e077/lang#development

> If you want to add new functionality for Jitsi Meet and you have texts that need to be translated please use our translation module. It is located in modules/translation. You must add key and value in main.json file in English for each translatable text. Than you can use the key to get the translated text for the current language.
> 
> WARNING: Please don't change the other language files except main.json! They must be updated and translated via our Pootle user interface!

Comment (guerda) [reply react/features/welcome/components/WelcomePage.web.js]: Thanks for the hint! I added a label and a translation accordingly.

Comment (guerda) [reply react/features/welcome/components/WelcomePage.web.js]: Hi @luixxiul ! Would it be possible for you to re-review the change? I added the translation as you proposed.
Thanks!

Comment (luixxiul) [reply react/features/welcome/components/WelcomePage.web.js]: Thanks for the ping!

While I do not have the authority to merge the PR as I'm just a contributor, I am going to check the PR to see if the change should work as expected 😄

**Comment by luixxiul** [comment]
It looks working, yet I am not quite sure if this is proper from the viewpoint of design... 🤔 

Clipboard02

**Comment by saghul** [comment]
We need to check with the design team to see if this is something we want to pursue.

**Comment by luixxiul** [comment]
Does applying `aria-label` to `<input>` not work? ie.

````
<input
     accessibilityLabel = " { t('welcomepage.roomLabel') }"
    ....
````

cf:
https://github.com/jitsi/jitsi-meet/blob/0598e7369b2dfac9c11637c3c356cf634466430f/react/features/toolbox/components/web/Toolbox.js#L1101

**Comment by luixxiul** [comment]
@saghul did you check-in with the design team yet? https://github.com/jitsi/jitsi-meet/pull/5740#issuecomment-635172244

**Comment by saghul** [comment]
Sorry, I hadn’t had a chance yet.

**Comment by ptrin** [comment]
As in this comment https://github.com/jitsi/jitsi-meet/pull/5740#issuecomment-635173713 it is possible to improve the accessibility of this field by using aria-label without showing a visible label, so this shouldn't require design input.

**Comment by guerda** [comment]
Thanks @ptrin & @luixxiul for the suggestion! I changed the label accordingly, so that it should be hidden, but still available for screen readers.
Review and feedback is appreciated!

**Comment by Echolon** [comment]
@saghul reminder to review

**Pull request closed without merging**